src/System.Security.Permissions/src/System/Security/CodeAccessPermission.cs

```diff
@@ -0,0 +1,20 @@
+namespace System.Security
+{
+    public abstract partial class CodeAccessPermission : System.Security.IPermission, System.Security.ISecurityEncodable, System.Security.IStackWalk
+    {
+        protected CodeAccessPermission() { }
+        public void Assert() { }
+        public abstract System.Security.IPermission Copy();
+        public void Demand() { }
+        public void Deny() { }
+        public override bool Equals(object obj) { return default(bool); }
+        //    public abstract void FromXml(System.Security.SecurityElement elem);
+        public override int GetHashCode() { return default(int); }
+        public abstract System.Security.IPermission Intersect(System.Security.IPermission target);
+        public abstract bool IsSubsetOf(System.Security.IPermission target);
+        public void PermitOnly() { }
```

Comment:
More specifically, throwing from PermitOnly (along with Deny) is how we can ensure that code won't depend on permissions for security (while NOPing Demand).